Documentation/WritingDocForExtension/Webhook.rst

```diff
@@ -15,34 +15,33 @@ following hosters:
   - :ref:`webhook-bitbucket-cloud` and Bitbucket self-hosted
   - :ref:`GitLab Cloud <webhook-gitlab>` and :ref:`GitLab self-hosted <webhook-gitlab>`
 
-If none of the above is used, a mirror to one of these is recommend. Also the
+If none of the above is used, a mirror to one of these is recommended. Also the
 `Documentation Team <https://typo3.org/community/teams/documentation/>`__ can
 setup custom hostings.
 
+The following sections provide illustrated walkthrough tutorials on how to add
+the necessary hook for the respective hoster.
+
 .. important::
 
-   The first time a new repository is triggering the webhook, it will not be
-   processed. Instead the TYPO3 Documentation Team has to approve. This is
-   necessary in order to prevent miss usage of the infrastructure and extension
+   The first time a new repository triggers the webhook, the request will not be
+   processed. Instead, the TYPO3 Documentation Team has to approve. This is
+   necessary to prevent misuse of the infrastructure and extension
    names.
 
-   In case this takes too long, request the approval within Slack channel
+   In case this takes too long, request the approval within the Slack channel
    `#typo3-documentation <https://typo3.slack.com/messages/C028JEPJL>`_.
    Registration for Slack is available at `my.typo3.org
    <https://my.typo3.org/index.php?id=35>`__.
 
-The following chapters provide illustrated walkthrough tutorials on how to add
-the necessary hook in those systems.
-
 .. _webhook-how-webhook-works:
 
-How webhook works
-=================
+How webhooks works
```

Comment:
if webhooks is plural then "works" needs to be "work"

Reply:
right. I'll fix it.